database/sqlite/sqlite_aclk_alert.c

```diff
@@ -486,17 +456,49 @@ void sql_queue_existing_alerts_to_aclk(RRDHOST *host)
     char uuid_str[UUID_STR_LEN];
     uuid_unparse_lower_fix(&host->host_uuid, uuid_str);
     BUFFER *sql = buffer_create(1024, &netdata_buffers_statistics.buffers_sqlite);
+    sqlite3_stmt *res = NULL;
+    int rc;
 
-    buffer_sprintf(sql,"delete from aclk_alert_%s; " \
-                       "insert into aclk_alert_%s (alert_unique_id, date_created, filtered_alert_unique_id) " \
-                       "select unique_id alert_unique_id, unixepoch(), unique_id alert_unique_id from health_log_%s " \
-                       "where new_status <> 0 and new_status <> -2 and config_hash_id is not null and updated_by_id = 0 " \
-                       "order by unique_id asc on conflict (alert_unique_id) do nothing;", uuid_str, uuid_str, uuid_str);
+    netdata_rwlock_wrlock(&host->health_log.alarm_log_rwlock);
 
-    netdata_rwlock_rdlock(&host->health_log.alarm_log_rwlock);
+    buffer_sprintf(sql, "delete from aclk_alert_%s; ", uuid_str);
```

Comment:
I already read papers saying that `delete` is faster than `truncate`, on the other hand some databases when we delete the data is still kept hidden, so few questions:

- Did we benchmark `delete` vs`truncate` on SQLite? (I am ignoring `drop`, because it should be more expensive)
- considering that we are deleting `aclk_alert` content, I understand we are not unifying them, do we plan to unfy in the future?
- Are we going to run `vacuum` on these tables?

Reply:
There is no truncate but there is a special delete optimization for this case. 
```
When the WHERE clause and RETURNING clause are both omitted from 
a DELETE statement and the table being deleted has no triggers, SQLite 
uses an optimization to erase the entire table content without having to visit 
each row of the table individually. 
```

Reply:
Thank you again! :)
The Lite is really light :laughing: